core/dbt/flags.py

```diff
@@ -39,7 +39,6 @@
 WHICH = None
 INDIRECT_SELECTION = None
 LOG_CACHE_EVENTS = None
-EVENT_BUFFER_SIZE = 100000
```

Comment:
@jtcohen6 is this flag considered experimental?  Does it need to be flagged as deprecated without being removed entirely.  It obviously would no longer do anything but if we remove it as a flag entirely, it causes an error for anyone using it as a parameter.

Reply:
I don't think anybody would have actually used the event history buffer because prior to the protobuf work it totally lacked all useful logging information (such as the timestamp, level, invocation_id, thread, etc). So in my opinion we shouldn't have to go through a deprecation cycle.

Reply:
I brought it up because I remember an issue when someone in the community was asking about it.  An issue I cannot find so I'm possibly mistaken.

Reply:
You do dream about these things, right? ;-)

Reply:
We did document this: https://docs.getdbt.com/reference/events-logging#python-interface

But I don't think anyone was really using this, so let's remove it entirely for now. We can add it back with a deprecation cycle if we encounter significant pushback/outcry.